modules/installation-initializing.adoc

```diff
@@ -344,7 +347,7 @@ ifdef::openshift-origin[]
 This field is optional.
 endif::[]
 endif::rhv[]
-ifeval::["{context}" == "installing-gcp-user-infra"]
+ifdef::gcp,azure-user-infra[]
```

Comment:
Unfortunately for both of us, I'm not an installation guru.  That said, several `ifeval` checks set the `gcp` attribute.  More than just the original `installing-gcp-user-infra` context.

I'm just a second set of eyes that's letting you know that you might be revealing this content in additional contexts.  Because the PR mentions Azure and not GCP, I'm pausing to make sure that _you're_ sure.

Reply:
The blurb in question appears in both gcp and azure content. It was originally in gcp and I added it to azure.